src/components/UpdatePaymentDetailsPopup/Steps/FinalizeAddPaymentDetails.js

```
@@ -33,6 +34,7 @@ const FinalizeAddPaymentDetails = () => {
           })
         );
         dispatch(fetchPaymentDetails());
+        dispatch(setGracePeriodError(false));
```

was this decision (to hide the error banner temporarily,  when the payment method was updated) approved by the product team?

---

Nope, will ask then